shared-libs/message-utils/package.json

```diff
@@ -8,16 +8,17 @@
   "author": "",
   "license": "Apache-2.0",
   "devDependencies": {
-    "@medic/phone-number": "1.0.0",
-    "bikram-sambat-bootstrap": "^1.4.2",
     "chai": "^4.2.0",
+    "mocha": "^5.2.0"
+  },
+  "dependencies": {
+    "@medic/phone-number": "file:../phone-number",
+    "bikram-sambat-bootstrap": "^1.4.2",
+    "google-libphonenumber": "^3.2.1",
     "gsm": "^0.1.4",
-    "mocha": "^5.2.0",
     "moment": "^2.22.2",
     "mustache": "^3.0.1",
-    "object-path": "^0.11.4"
-  },
-  "dependencies": {
+    "object-path": "^0.11.4",
```

Comment:
This has moved most of the libs from devDependencies to dependencies - were they previously in the wrong place?

Reply:
Yep. Because I'm installing them with `--production`, I needed them in the proper place (they're not dev dependencies).

Reply:
Awesome.